WordPress/WordPressTest/JetpackFeaturesRemovalCoordinatorTests.swift

```diff
@@ -346,3 +346,13 @@ private extension Date {
                                                from: self)
     }
 }
+
+internal extension RemoteFeatureFlagStore {
+    func update(using remote: FeatureFlagRemote, waitOn test: XCTestCase) {
+        let exp = test.expectation(description: "Store finishes update")
+        update(using: remote) {
+            exp.fulfill()
+        }
+        test.wait(for: [exp], timeout: 1)
+    }
+}
```

Comment:
Interesting approach. For a moment I got worried because it looked like the test was no longer testing application code, but then I looked at the implementation and realized it called the app code internally.

This has definitely value because it DRYes what would otherwise be a lot of repeated async expectations.

I wonder if the DRY could be pushed even further. Most tests have a similar arrange structure:

```swift
        let store = ...
        let flags = ...
        let remote = ...
        store.update(using: remote, waitOn: self)
```